# HG changeset patch
# User Kai T. Ohlhus <k.ohlhus@gmail.com>
# Date 1423812288 -3600
#      Fri Feb 13 08:24:48 2015 +0100
# Node ID 80462dbfb3ac26f443d88f310ed4d3d0f140b5e1
# Parent  a81177f4bfe6bd438781da473a2d02aaa0ff6f81
doc: NEWS entry for changes in the "ismatrix" function (bug #42422).

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -35,16 +35,20 @@ Summary of important user-visible change
     disable_range   disable_diagonal_matrix   disable_permutation_matrix
 
     All three optimizations are disabled if Octave is started with the
     --braindead command line option.
 
  ** For compatibility with Matlab, the "backtrace" warning option is now
     enabled by default.
 
+ ** For compatibility with Matlab, the "ismatrix(x)" function now only checks
+    the dimension of "x". The old behaviour of "ismatrix" is obtained by
+    "isnumeric (x) || islogical (x) || ischar (x)".
+
  ** The preference
 
       do_braindead_shortcircuit_evaluation
 
     is now enabled by default.
 
  ** The preference
 
